Demo直接Crash了

点击启动插件，直接Crash了，信息如下

```
2019-12-03 19:17:39.877 24400-24584/com.tencent.shadow.sample.host E/AndroidRuntime: FATAL EXCEPTION: pool-3-thread-1
    Process: com.tencent.shadow.sample.host, PID: 24400
    java.lang.RuntimeException: android.os.DeadObjectException
        at com.tencent.shadow.sample.manager.SamplePluginManager$1.run(SamplePluginManager.java:118)
        at java.util.concurrent.ThreadPoolExecutor.processTask(ThreadPoolExecutor.java:1187)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1152)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
        at java.lang.Thread.run(Thread.java:784)
     Caused by: android.os.DeadObjectException
        at android.os.BinderProxy.transactNative(Native Method)
        at android.os.BinderProxy.transact(Binder.java:1147)
        at com.tencent.shadow.dynamic.manager.BinderPluginLoader.loadPlugin(BinderPluginLoader.java:46)
        at com.tencent.shadow.sample.manager.FastPluginManager.loadPlugin(FastPluginManager.java:141)
        at com.tencent.shadow.sample.manager.FastPluginManager.convertActivityIntent(FastPluginManager.java:119)
        at com.tencent.shadow.sample.manager.FastPluginManager.startPluginActivity(FastPluginManager.java:111)
        at com.tencent.shadow.sample.manager.SamplePluginManager$1.run(SamplePluginManager.java:116)
        at java.util.concurrent.ThreadPoolExecutor.processTask(ThreadPoolExecutor.java:1187) 
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1152) 
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641) 
        at java.lang.Thread.run(Thread.java:784) 

```

--- a/projects/sample/source/sample-host/src/main/java/com/tencent/shadow/sample/host/HostApplication.java
+++ b/projects/sample/source/sample-host/src/main/java/com/tencent/shadow/sample/host/HostApplication.java
@@ -65,7 +65,6 @@ public class HostApplication extends Application {
             return;
         }
         StrictMode.VmPolicy.Builder builder = new StrictMode.VmPolicy.Builder();
-        builder.penaltyDeath();
         builder.detectNonSdkApiUsage();
         StrictMode.setVmPolicy(builder.build());
     }

--- a/projects/test/dynamic/host/test-dynamic-host/src/main/java/com/tencent/shadow/test/dynamic/host/HostApplication.java
+++ b/projects/test/dynamic/host/test-dynamic-host/src/main/java/com/tencent/shadow/test/dynamic/host/HostApplication.java
@@ -69,7 +69,6 @@ public class HostApplication extends Application {
             return;
         }
         StrictMode.VmPolicy.Builder builder = new StrictMode.VmPolicy.Builder();
-        builder.penaltyDeath();
         builder.detectNonSdkApiUsage();
         StrictMode.setVmPolicy(builder.build());
     }

--- a/projects/test/none-dynamic/host/test-none-dynamic-host/src/main/java/com/tencent/shadow/test/none_dynamic/host/HostApplication.java
+++ b/projects/test/none-dynamic/host/test-none-dynamic-host/src/main/java/com/tencent/shadow/test/none_dynamic/host/HostApplication.java
@@ -115,7 +115,6 @@ public class HostApplication extends Application {
             return;
         }
         StrictMode.VmPolicy.Builder builder = new StrictMode.VmPolicy.Builder();
-        builder.penaltyDeath();
         builder.detectNonSdkApiUsage();
         StrictMode.setVmPolicy(builder.build());
     }

